doozer/doozerlib/cli/images_streams.py

```diff
@@ -796,6 +796,7 @@ def search_issues(query):
         fields = {
             'project': {'key': project},
             'issuetype': {'name': 'Bug'},
+            'labels': ["art:reconciliation", f"art:distgit:{distgit_key}"],
```

Comment:
this is petty but good style is to use consistent quoting
```suggestion
            'labels': ['art:reconciliation', f'art:distgit:{distgit_key}'],
```
(good style is also to use double quotes, not single, but that's more controversial)